src/app/pages/send/send-crypto-asset-form/family/bitcoin/hooks/use-calculate-max-spend.ts

```diff
@@ -0,0 +1,32 @@
+// To know a users max spendable amount, we need to consider how many UTXOs they
+import { useMemo } from 'react';
+
+import { createMoney } from '@shared/models/money.model';
+
+import { useGetUtxosByAddressQuery } from '@app/query/bitcoin/address/utxos-by-address.query';
+import { useBitcoinFeeRatesInVbytes } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { useCurrentBtcAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/bitcoin-account.hooks';
+
+import { BtcSizeFeeEstimator } from '../fees/tx-size-calculator';
+
+// have, in order to figure out the transaction size and the current fee rates
+// ts-unused-exports:disable-next-line
+export function useCalculateMaxSpend() {
+  const currentAccountBtcAddress = useCurrentBtcAccountAddressIndexZero();
+  const { data: utxos } = useGetUtxosByAddressQuery(currentAccountBtcAddress);
+  const { data: feeRate } = useBitcoinFeeRatesInVbytes();
```

Comment:
Isn't it more accurate to say `..SatPerVbytes` for the fee rate? I feel like it is typ written in docs as `(in sat/vB)`

Reply:
Nvm, I see that is coming from the new api now.